services/streammanager/common/common.go

```diff
@@ -0,0 +1,49 @@
+package common
```

Comment:
It is added to avoid import cycle problem as streammanger package depends on destination specific managers so destination managers shouldn't import streammanager.